content/id/docs/concepts/workloads/pods/pod-preset.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Pod Preset
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+Halaman ini menyajikan sebuah gambaran umum dari `Pod Preset`, yang merupakan objek untuk memasukkan informasi tertentu ke dalam `Pod` pada saat waktu penciptaan. Informasi dapat berupa *secret*, *volume*, *volume mount*, dan variabel *environment*.
+{{% /capture %}}
+
+{{% capture body %}}
+## Memahami *Pod Preset*
+---
+
+Sebuah *Pod Preset* adalah sebuah *resource* API untuk memasukkan kebutuhan *runtime* tambahan ke dalam sebuah *Pod* pada saat waktu penciptaan. Kamu akan menggunakan *label selector* untuk menunjuk *Pod-pod* dimana *Pod Preset* diterapkan.
```

Comment:
Nit: use `**` for bold and `_` for italic

Reply:
fixed in https://github.com/kubernetes/website/pull/14394/commits/5e8fb53a826b1119b77b923408a4f7a9e64e6415